quantumdb-core/src/main/java/io/quantumdb/core/backends/Backend.java

```diff
@@ -21,10 +22,11 @@
 	 * Persists the current state of the database schema and its evolution to the database.
 	 *
 	 * @param state The current state of the database schema.
+	 * @param stage The stage that QuantumDB might want to save. Can be null.
 	 *
 	 * @throws SQLException In case the database could not be reached, or queried correctly.
 	 */
-	void persistState(State state) throws SQLException;
+	void persistState(State state, Stage stage) throws SQLException;
```

Comment:
When performing data changes, QuantumDB wants to persist the database state.

However, I do not save any changeset/changelog to the database that is past the last active version. With the Stage object I can request the last version that the migration performed.